extensions/sample-profile/src/main/java/com/liferay/extensions/sample/profile/OverriddenArgs.java

```diff
@@ -16,10 +16,13 @@
 
 package com.liferay.extensions.sample.profile;
 
+import com.beust.jcommander.Parameters;
+
 import com.liferay.blade.cli.command.DeployArgs;
 
 /**
  * @author Liferay
  */
+@Parameters(commandDescription = "Overriden Deploy Command", commandNames = "deploy")
```

Comment:
I see, we need to re-add the parameters.  Good work.

Reply:
Thanks :)